source/addComment.js

```diff
@@ -1,3 +1,22 @@
 $(document).ready(function(){
-  // add your code here
+  $('#comments #new_comment_button').on('click', function(e) {
+    e.preventDefault();
+    var newCommentButton = $('#comments #new_comment_button');
+    newCommentButton.detach(); 
```

Comment:
I haven't seen the `detach()` method before. Cool usage. I think it may have been more useful in the line before though.